BackendTests/UnitTests/MetAPIServiceTests.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Net.Http;
+using Backend.API.Services;
+using Moq;
+using Xunit;
+
+namespace BackendTests.UnitTests
+{
+    [SuppressMessage("Rule Category", "CA5394", Justification = "No security threat on data mocking.")]
+    public class MetAPIServiceTests
+    {
+        private readonly float _lat;
+        private readonly float _lon;
+        private readonly MetAPIService _metAPIService;
+        private readonly MetAPIServiceMocked _metAPIServiceMocked;
+        private readonly Random _random;
+
+
+        public MetAPIServiceTests()
+        {
+            _metAPIServiceMocked = new MetAPIServiceMocked();
+            _random = new Random();
+            _lon = (float) _random.NextDouble() * (63.433f - 63.42f) + 63.42f;
+            _lat = (float) _random.NextDouble() * (10.4f - 10.38f) + 10.38f;
```

Comment:
```suggestion
            _lon = 63.42;
            _lat = 10.38;
```

Simplify the test by using hard-coded variables.

Alternatively, you could perform some [property based testing](https://medium.com/criteo-engineering/introduction-to-property-based-testing-f5236229d237) by having lat and lon be generated in the respective accepted ranges (-180->180 and -90->90).

Reply:
Changed